configure.ac

```diff
@@ -64,6 +77,17 @@ if [[ -f "$header_prefix/include/libzfs/libzfs.h" ]] || [[ "$zof" = true ]]; the
 "
 	LDFLAGS="-L${header_prefix}/lib ${LDFLAGS}"
 
+elif [[ "$build_macos" = "yes" ]] && [[ "x$with_o3x" != "x"  ]]; then
+	zof=true
```

Comment:
I just noticed `zof=true`, `zof` means `zfs_on_freebsd` which this is clearly not. Can you please confirm if this is intentional ? If yes, please add a new variable perhaps as this will cause confusion. And I am not sure if this was there before 🤔 Maybe got in recent commits ?

Reply:
Yeah, probably should rename this to `o3x` and use it wherever `zof` was used.